src/mds/MDSRank.cc

```diff
@@ -2782,7 +2792,12 @@ void MDSRankDispatcher::handle_asok_command(
 	}));
     return;
   } else if (command == "scrub pause") {
-    if (whoami != 0) {
+    if (!is_active()) {
+      *css << "MDS is not active";
+      r = -CEPHFS_EINVAL;
+      goto out;
+    }
+    else if (whoami != 0) {
```

Comment:
this is a boilerplate code; use something like
```
if(command.contains("scrub") && !is_active()) {
	*css << "MDS is not active";
    r = -CEPHFS_EINVAL;
    goto out;
}
```
in `MDSRankDispatcher::handle_asok_command` to eliminate the repetition

Reply:
I'm a bit wary of future work that _might_ introduce "scrub" keyword o_O

Reply:
We are not checking `is_active()` for `scrub status` command, and the above suggestion will cover it also. And I guess we won't want to disable status command too in this case, as there may be cases where we need to check the status of scrub. or we can add one more check there like `.. && command != "scrub status" ` . But I will like to get this thing clear before that if we want this check for `scrub status` or not? @mchangir @vshankar

Reply:
> We are not checking `is_active()` for `scrub status` command, and the above suggestion will cover it also. And I guess we won't want to disable status command too in this case, as there may be cases where we need to check the status of scrub. or we can append the check like `.. && command != "scrub status" ` . But I will like to get this thing clear before that if we want this check for `scrub status` or not? @mchangir @vshankar

scrub status is supposed to be available only with mds rank 0
so, applying the same boilerplate check is good for scrub status as well